draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1761,29 +1761,37 @@ interest, or both. For example, a monitor may be configured to report on all
 certificates that apply to a specific domain name when fetching new entries for
 consistency validation.
 
-A monitor needs to, at least, inspect every new entry in each log it watches.
-It may also want to keep copies of entire logs. In order to do this, it should
-follow these steps for each log:
+A monitor MUST at least inspect every new entry in every log it watches, and it
+MAY also choose to keep copies of entire logs.
+
+To inspect all of the existing entries, the monitor SHOULD follow these steps
+once for each log:
 
 1. Fetch the current STH ({{get-sth}}).
 
 2. Verify the STH signature.
 
-3. Fetch all the entries in the tree corresponding to the STH
-   ({{get-entries}}).
+3. Fetch all the entries in the tree corresponding to the STH ({{get-entries}}).
+
+4. If applicable, check each entry to see if it's a certificate of interest.
 
-4. Confirm that the tree made from the fetched entries produces the same hash as
+5. Confirm that the tree made from the fetched entries produces the same hash as
    that in the STH.
 
-5. Fetch the current STH ({{get-sth}}). Repeat until the STH changes.
+To inspect new entries, the monitor SHOULD follow these steps repeatedly for
+each log:
 
-6. Verify the STH signature.
+6. Fetch the current STH ({{get-sth}}). Repeat until the STH changes.
```

Comment:
nit: Re-start numbering from 1? it's not really a part of the same process.

Reply:
I agree it's not part of the same process.  However, if I were to restart the numbering then "11. Repeat from step 6" would become "6. Repeat from step 1" and there would be two step 1s on the page.  Which step 1 would it be referring to?
I'd rather not restart the numbering simply so that this ambiguity is avoided.

Reply:
Fine by me.